src/safeds/data/tabular/transformation/_k_nearest_neighbors_imputer.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from numpy import nan
```

Comment:
For performance reasons, avoid top-level imports of third-party libraries. Due to the eager import system of Python, a simple import like this can take several seconds. To emulate lazy imports, move the import into the function that needs the `nan` (here `__init__`).